js/classes/TinyMCEHelper.js

```diff
@@ -10,8 +10,7 @@
  *
  * @brief TinyMCE helper methods
  */
-(function($) {
-
+(function ($) {
```

Comment:
Hmm, I'm surprised the JS linter didn't complain about this -- it's really, really picky about whitespacing.